hal/inc/concurrent_hal.h

```diff
@@ -186,7 +186,7 @@ int os_semaphore_take(os_semaphore_t semaphore, system_tick_t timeout, bool rese
 int os_semaphore_give(os_semaphore_t semaphore, bool reserved);
 
 #define _GLIBCXX_HAS_GTHREADS
-#include <bits/gthr.h>
+//#include <bits/gthr.h>
```

Comment:
I'm guessing this didn't build for you? We need this for the multithreading build, so please re-insert this line.